addons/netfox.extras/window-tiler.gd

```diff
@@ -0,0 +1,61 @@
+extends Node
+
+
+func _ready() -> void:
+	EngineDebugger.register_message_capture("tile-session", _on_tile_session)
+	EngineDebugger.send_message.call_deferred("tile-session:get_id", [])
+
+
+func _on_tile_session(message: String, data: Array) -> bool:
+	if message == "session_id":
+		var info: Dictionary = data[0]
+		var id = info.get("id", -1)
+		var total = info.get("total", 0)
+		if total > 1:
+			for i in range(total):
+				if i == id:
+					tile_window(i, total)
+		return true
+	return false
+
+
+func tile_window(i: int, total: int) -> void:
+
+	if not ProjectSettings.get_setting("netfox/extras/auto_tile_windows", true):
+		return
+
+	var screen = ProjectSettings.get_setting("netfox/extras/screen", 0)
+	var screen_rect = DisplayServer.screen_get_usable_rect(screen)
+
+	var window: Window = get_tree().get_root()
+	window.set_current_screen(screen)
+	window.borderless = ProjectSettings.get_setting("netfox/extras/borderless", true)
```

Comment:
There might be some issue on Linux - when borderless is active, the window positions and sizes are ignored and everything is just pushed  to the middle of the desktop ( i.e. the center of all screens combined, assuming this is related to X11 ). 

I'll check if there's any Godot engine issue for this.

Reply:
Works fine in KDE Plasma with X11 but if its flaky I can remove it.

Main reason I included it was the window sizes given are the actual rendering portion so you get overlap with the title bars when tiling.